.github/workflows/platform.yml

```diff
@@ -17,6 +17,16 @@ env:
 # TODO: upstream jobs
 
 jobs:
+  bun-pack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - uses: ./.github/actions/node/active-lts
+      - run: mkdir -p npm bun
+      - run: FILENAME=$(npm pack) && tar -zxf $FILENAME -C npm
+      - run: bun pm pack --filename bun.tgz && tar -zxf bun.tgz -C bun
+      - run: diff -r npm bun
```

Comment:
This will diff the filenames + their content. Do we really need to also diff the content? We could do this whole operation faster if we just differed that filenames contained in the two tarballs 🤔 

The the whole operation could be:

```sh
bun pm pack --filename bun.tgz
diff <(tar -tf $(npm pack) | sort) <(tar -tf bin.tgz | sort)
```

Reply:
We don't necessarily need to _diff_ the content, but we should at least ensure that they're the same. Maybe a quick hash would be better?

Reply:
Taking a look at the job run `diff` takes less than 1 second to run so I think it's fine. Does it do more than we need? Maybe or maybe not, but it's fast enough that it shouldn't matter.